pkg/distro/fedora/images.go

```diff
@@ -333,7 +331,7 @@ func liveInstallerImage(workload workload.Workload,
 	img.Variant = "Workstation"
 	img.OSVersion = d.osVersion
 	img.Release = fmt.Sprintf("%s %s", d.product, d.osVersion)
-	img.ISOLabel = fmt.Sprintf(ISO_LABEL, img.Product, img.OSVersion, img.Variant, img.Platform.GetArch())
+	img.ISOLabel = fmt.Sprintf(t.arch.distro.ISOLabel(), img.Product, img.OSVersion, img.Variant, img.Platform.GetArch())
```

Comment:
This seems awkward. distro.ISOLabel() is the template, so it is really what should decide what info from img it wants to use to populate it instead of passing it all in the right order. But passing img to it also seems wrong :)
The iso label has a stronger relation to the image type than the distro so maybe it would make sense to move the template and string creation into there and add a function to populate the string, or get it when needed?

Reply:
I mostly followed the style that we already have for OSTree ref template https://github.com/osbuild/images/blob/5538c287fad5c16f6cc9fbf8afecc73450164b79/pkg/distro/distro.go#L52-L53

I'm open to suggestions and don't have strong preference here. The reality in RHEL distros has been so far that we had the template defined on distro level. For Fedora, the template was also technically the same for the whole distro.

I think that the problem here is that the method name suggests that it will return the actual label and not the template. I can rename it.

Reply:
It's not just the name, it's using a template from a place that doesn't have the info to populate it with. The template should be next to the img.Product, etc. so that it can be changed if needed. Really, it ought to be a function that is called to get the ISOLabel -- since I also see there is code checking to see if the string has been set, overriding yet another template.

Reply:
So, that was actually introduced by https://github.com/osbuild/images/pull/488.

In Fedora land, this is simple, but it gets complicated in RHEL due to reusing the same image type definitions for RHEL and CS.

I now understand that my PR does not really resolves https://github.com/osbuild/images/issues/491, it is keeping the status quo and just making it possible to get the template value from the `distro.Distro`, so that it becomes obtainable from `rhel_common` structs in other packages.

I'll think about this a bit.

Reply:
I've submitted https://github.com/osbuild/images/pull/542

Reply:
This code change is no longer part of this PR, thus resolving.